### ISSUE TITLE
One-click muting

### ISSUE THREAD

**Issue by RomanHauksson** [insin/control-panel-for-twitter#470]
It would be nice if I could enable a button to mute the author of a Tweet with one click.

Right now, whenever I want to mute a user, I need to click the three-dot menu on the Tweet and then click Mute. Instead, I'd like that button to exist directly on the Tweet itself, e.g. next to the Bookmark button.

**Comment by eyupa** [comment]
You're absolutely right, some extensions do that, but I don't want to install another plugin for it. 
Mute, block and spam buttons should be add
